src/components/ListItem/ListItem.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import ListItem from './ListItem';
+
+test('renders learn react link', () => {
+  render(<ListItem/>);
+  const Item = screen.getByTestId('TestItem');
```

Comment:
It could be better to check the outermost container in render method. That way you don't need to change the render test case in case the implementation changes